Gemfile

```diff
@@ -30,6 +30,7 @@ gem "acts_as_tree",                   "~>2.1.0" # acts_as_tree needs to be requi
 gem "ancestry",                       "~>2.2.1",       :require => false
 gem "ansible_tower_client",           "~>0.4.1",       :require => false
 gem "aws-sdk",                        "~>2",           :require => false
+gem "bundler",                        ">=1.11.1",      :require => false
```

Comment:
We've had issues with bundler processing dependencies in the past and we've set a minimum version in various places.  This seems like the natural place to state the dependency.